Bottom: a365a32d95f0a33b2bff8db2699286f3f75947f3
Top:    567ae58df6b3e81a7de5d078ffa8d52ff6966350
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:41:57 +0000

Add reversed orientation field to face3d structure


---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 4838c43..95232a6 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
